client/my-sites/themes/thanks-modal.jsx

```diff
@@ -81,71 +77,47 @@ const ThanksModal = React.createClass( {
 					{ this.props.source === 'list' ? features : customize }
 				</li>
 			<li>
-				{ translate( 'Have questions? Stop by our {{a}}support forums.{{/a}}', {
-					components: {
-						a: <a href={ this.props.forumUrl }
-							onClick={ this.onLinkClick( 'support' ) } />
-					}
-				} ) }
+				{ this.renderSupportInfo() }
 			</li>
 			</ul>
 		);
 	},
 
-	renderWporgThemeInfo( themeUri ) {
-		if ( themeUri ) {
-			return (
-				<li>
-					{ translate( 'Learn more about this {{a}}awesome theme{{/a}}.', {
-						components: {
-							a: <a href={ themeUri }
-								onClick={ this.onLinkClick( 'org theme' ) } />
-						}
-					} ) }
-				</li>
-			);
-		}
+	renderThemeInfo() {
+		return translate( '{{a}}Learn more about{{/a}} this theme.', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **22** times:
`translate( 'Learn more about this {{a}}awesome theme{{/a}}.' )` *ES Score: 9.68*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).